sdk/core/azure-core-http-vertx/src/main/java/com/azure/core/http/vertx/implementation/VertxRequestWriteSubscriber.java

```diff
@@ -122,19 +130,50 @@ private void onErrorInternal(Throwable throwable) {
         // code 2 and greater are completion states which means the error should be dropped as we already completed.
         if (state.code >= 2) {
             Operators.onErrorDropped(throwable, Context.of(contextView));
+
+            // Also, even though Reactor may have an operator for the dropped error or will be logged by Operators
+            // itself, we should log as well as at least this will help associate the error with this class.
+            LOGGER.atError().log(() -> "VertxRequestWriteSubscriber dropped an exception as it already reached a "
```

Comment:
similar to previous comment: does it need to be an error? Would it result in terminal exception bubbling up to the user code?

Also, I don't think you need lambda here, compiler should optimize string literal concat

Reply:
This is the only API that takes Throwable